packages/framework/tests/Unit/Component/ClassExtension/DocBlockParserTest.php

```diff
@@ -0,0 +1,163 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\FrameworkBundle\Unit\Component\ClassExtension;
+
+use phpDocumentor\Reflection\Fqsen;
+use phpDocumentor\Reflection\Types\Array_;
+use phpDocumentor\Reflection\Types\Compound;
+use phpDocumentor\Reflection\Types\Integer;
+use phpDocumentor\Reflection\Types\Null_;
+use phpDocumentor\Reflection\Types\Object_;
+use PHPUnit\Framework\TestCase;
+use Roave\BetterReflection\Reflection\ReflectionObject;
+use Roave\BetterReflection\Reflection\ReflectionParameter;
+use Roave\BetterReflection\Reflection\ReflectionProperty;
+use Shopsys\FrameworkBundle\Component\ClassExtension\DocBlockParser;
+use Tests\FrameworkBundle\Unit\Component\ClassExtension\Source\DummyClassForAnnotationsReplacerTest;
+
+class DocBlockParserTest extends TestCase
+{
+    /**
+     * @var \Shopsys\FrameworkBundle\Component\ClassExtension\DocBlockParser
+     */
+    private DocBlockParser $docBlockParser;
+
+    protected function setUp(): void
+    {
+        $this->docBlockParser = new DocBlockParser();
```

Comment:
just a nitpick - I do not insist on changing this, I am ok with the current solution, however, we use https://packagist.org/packages/zalas/phpunit-injector so you could just add `@inject` annotation to the `$docBlockParser` property and you would not need the `setUp` method at all. Or do you need a new instance of `DocBlockParser` to be created before each of the tests?

Reply:
Ah, I did not notice it is a unit test, and what is more important, I did not know it is not possible to use in unit tests :smile: Thanks for the info :+1: